.github/workflows/ci.yml

```diff
@@ -166,6 +166,7 @@ jobs:
     - name: Install requirements
       run: |
         . .venv/bin/activate
+        pip install "allennlp[all] @ git+https://github.com/allenai/allennlp.git@main"
```

Comment:
I think we want to change this in `requirements.txt` instead of here.

Reply:
We don't want the release of -models to depend on the main branch of allennlp though.

Reply:
We already handle that with the `ALLENNLP_VERSION_OVERRIDES` env variable

Reply:
For development, I always have to change that line in `requirements.txt` to just say `allennlp`, so that `pip install -e ./allennlp -e ./allennlp-models` works.

Reply:
https://github.com/allenai/allennlp-models#from-source

Reply:
I thought to do that too at first, but the models repo doesn't need checklist. Only the tests do.

Reply:
Oh, then you could set `ALLENNLP_VERSION_OVERRIDE` to that